src/Framework/Constraint/Constraint.php

```diff
@@ -92,9 +91,9 @@ public function count()
     /**
      * Throws an exception for the given compared value and test description
      *
-     * @param mixed                                          $other             Evaluated value or object.
-     * @param string                                         $description       Additional information about the test
-     * @param SebastianBergmann\Comparator\ComparisonFailure $comparisonFailure
+     * @param mixed             $other             Evaluated value or object.
+     * @param string            $description       Additional information about the test
+     * @param ComparisonFailure $comparisonFailure
```

Comment:
Class is already imported here, see [`Constraint.php#L17`](https://github.com/sebastianbergmann/phpunit/blob/b2e74111300b09f5a23e6213e8abbeac6536bb85/src/Framework/Constraint/Constraint.php#L17).